ff/src/biginteger/mod.rs

```diff
@@ -102,6 +102,10 @@ pub trait BigInteger:
     /// leading zeros.
     fn to_bits(&self) -> Vec<bool>;
 
+    /// Returns the byte representation in a big endian byte array, without
+    /// leading zeros.
```

Comment:
Is there a reason to omit leading zeros? It makes the size dependent on the value, which can be confusing. Also, `to_bits` is not value-dependent, AFAIK

Reply:
Oh you're right, `to_bits` isn't value dependent, so this is also not value dependent. (Its using the same bit iterator as `to_bits`) I'll fix the comments accordingly.